src/common/display/src/table_display.rs

```diff
@@ -140,17 +140,17 @@ pub fn make_comfy_table<S: AsRef<str>>(
                 .iter()
                 .map(|s| {
                     let mut str_val = s.str_value(i);
-                    if let Some(max_col_width) = max_col_width {
-                        if str_val.len() > max_col_width - DOTS.len() {
-                            str_val = format!(
-                                "{}{DOTS}",
-                                &str_val
-                                    .char_indices()
-                                    .take(max_col_width - DOTS.len())
-                                    .map(|(_, c)| c)
-                                    .collect::<String>()
-                            );
-                        }
+                    if let Some(max_col_width) = max_col_width
+                        && str_val.len() > max_col_width - DOTS.len()
+                    {
```

Comment:
What's with this weird formatting? Putting the `{` onto its own line is a waste of space -- is this something automatic @kevinzwang ? I'm wondering why it would come up here in this PR.

Reply:
It's because if-let chains were stabilized so the linter is making us use it now

Reply:
OOOOHHHHH okay. So is this from clippy?